app/assets/javascripts/host_edit.js

```diff
@@ -170,6 +171,10 @@ function submit_with_all_params() {
     url: $('form').attr('action'),
     data: serializeForm(),
     success: function(response) {
+      // workaround for redirecting to the new host details page
+      if (response.includes('layouts/react_application')) {
+        tfm.nav.pushUrl(`/new/hosts/${fqdn}`);
```

Comment:
oh, we can't use backticks in the legacy JS, that's why the tests are failing.
```suggestion
        tfm.nav.pushUrl('/new/hosts/' + fqdn);
```

Reply:
right! I just got used to es6 :)